src/main/java/uk/gov/hmcts/reform/professionalapi/service/impl/ProfessionalUserServiceImpl.java

```diff
@@ -76,6 +78,18 @@ public ProfessionalUser findProfessionalUserByEmailAddress(String email) {
         return user;
     }
 
+    public ProfessionalUser findProfessionalUserByEmailAddressForActiveOrganisation(String email) {

```

Comment:
We should not be having two duplicate service methods with different names. Please remove this and if you need to validation add it in the original method if possible, add an extra param if need be but never two analogous methods in the same interface.